client/actions/connectToDopplerActions.js

```diff
@@ -16,14 +16,24 @@ export function changeDopplerCredentialValiationStatus(valid = true) {
   };
 }
 
+export function userLoggedIn(dopplerAccountName) {
+  return {
+    type: types.USER_LOGGED_IN,
+    dopplerAccountName
+  };
+}
+
 export function connectToDoppler({ dopplerAccountName, dopplerApiKey }) {
   return (dispatch, getState) => {
     dispatch(connectingToDoppler(true));
     return appService
       .connectToDoppler({ dopplerAccountName, dopplerApiKey })
       .then(success => {
         dispatch(connectingToDoppler(false));
-        if (success) dispatch(push('/app/setup-doppler-list'));
+        if (success) {
+          dispatch(userLoggedIn(dopplerAccountName))
```

Comment:
I can't see the code where the doppler account name is saved, I'm missing something?

Reply:
check the file client/reducers/appSetupReducer.js in this PR.